app/models/building.rb

```diff
@@ -179,6 +179,36 @@ def self.ransackable_attributes(_auth_object = nil)
     super + %w[street_address]
   end
 
+  def self.as_geojson(buildings)
```

Comment:
I would make this a service object. I suppose an `as_geojson` instance method could convert a building model to a properties hash. But all the `record.respond_to?` bits make me wonder if it is a building model at all, and if it isn't, what is it doing here in the building model?